include/bluetooth/audio.h

```diff
@@ -36,13 +36,13 @@ struct bt_audio_cap;
 
 #define BT_BROADCAST_ID_SIZE             3 /* octets */
 
-#if defined(CONFIG_BT_BAP)
+#if defined(CONFIG_BT_BAP) && defined(CONFIG_BT_AUDIO_BROADCAST)
```

Comment:
I guess we eventually want to BAP and BROADCAST to depend on each other.  (One will depend upon the other, not sure now which one will be highest in the hierarchy.)
(That is outside the scope of this PR.)
When that is done, it will be sufficient to check for one of these.

(Ditto for endpoint.h)

Reply:
That is true. Currently they are separate entities. 

I think that in time we will get rid of the CONFIG_BT_BAP and simply use `CONFIG_BT_AUDIO_{UNICAST/BROADCAST}`.